core/config.py

```diff
@@ -61,6 +61,9 @@ class Configuration(ConfigurationConstants):
     # Environment variable for temporary reporting email
     REPORTING_EMAIL_ENVIRONMENT_VARIABLE = "SIMPLIFIED_REPORTING_EMAIL"
 
+    # Environment variable for collection manager name
+    CM_NAME_ENVIRONMENT_VARIABLE = "PALACE_CM_NAME"
```

Comment:
Minor comment on the name of this env variable. We might want to change this to either: 
- PALACE_NAME
- PALACE_REPORTING_NAME

I'm not sure having CM in the name adds much specificity to the variable.